vkdgen.py

```diff
@@ -47,47 +47,53 @@
 alias int32_t = int;
 alias int64_t = long;
 
-@nogc pure nothrow {
-	uint VK_MAKE_VERSION(uint major, uint minor, uint patch) {
+@nogc pure nothrow {{

```

Comment:
I chose to do substring replacement over Python's `format` function here so that you _didn't_ have to do this error-prone and tedious escaping of `{}`. Judging by one of your commits on your repo, this has already bitten you once.